__tests__/api/request_test.js

```diff
@@ -0,0 +1,36 @@
+let requests = require('../../api/requests');
+let moment = require('moment');
+jest.mock('googleapis');
+let google = require('googleapis');
+
+// This needs to be using a specific time 
+// Test delays means the times are sometimes out of sync
+let originalDate = moment("2013-08-26 16:55:00");
+let dateTime = originalDate.toISOString();
+let date = originalDate.format('YYYY-MM-DD');
```

Comment:
Could reuse the same variables from the mock file, just export them there

Reply:
Yup! Turned them into reusable variables and shared with request_test.js